tests/test_data/parameter_json_files/panda_test_parameters.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
Could: As I said in the previous review, would be nice to have a test that this can correctly trigger a panda scan

Reply:
The tests in `test_panda_flyscan_xray_centre_plan` now use these parameters